pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -571,6 +574,17 @@ func (s *DiscoveryServer) pushConnection(con *XdsConnection, pushEv *XdsEvent) e
 		} else {
 			adsLog.Debugf("Skipping push to %v, no updates required", con.ConID)
 		}
+
+		if s.StatusReporter != nil {
+			// this version of the config will never be distributed to this envoy because it is not a relevant diff.
+			// inform distribution status reporter that this connection has been updated, because it effectively has
+			go func() {
```

Comment:
Ithink we should remove the go routine. given we decided not to use it for registering 1 event, I am not sure it makes sense to use it for registering 4 events, unless there is some reason I am missing?

Reply:
Good Point.  Will do.